<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGui::ConnectionOptionPage</class>
 <widget class="QWidget" name="QtGui::ConnectionOptionPage">
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <property name="windowIcon">
   <iconset theme="network-connect">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="urlLabel">
     <property name="text">
      <string>Syncthing URL</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Widgets::ClearLineEdit" name="urlLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="authLabel">
     <property name="text">
      <string>Authentication</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="authCheckBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="userNameLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>User</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="Widgets::ClearLineEdit" name="userNameLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>disconnected</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="statusTextLabel">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QPushButton" name="connectPushButton">
     <property name="text">
      <string>Apply connection settings and try to reconnect</string>
     </property>
     <property name="icon">
      <iconset theme="view-refresh">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="Widgets::ClearLineEdit" name="apiKeyLineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="apiKeyLabel">
     <property name="text">
      <string>API key</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="insertFromConfigFilePushButton">
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Insert values from local Syncthing configuration</string>
     </property>
     <property name="icon">
      <iconset theme="edit-paste">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="Line" name="line">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widgets::ClearLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">qtutilities/widgets/clearlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>authCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>userNameLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>authCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>userNameLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>authCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>authCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
